### ISSUE TITLE
`imports.php` fails on reading tiger data for 2017 (while 2016 and 2015 work - format change?)

### ISSUE THREAD

**Issue by Fusl** [openstreetmap/Nominatim#817]
Hi,

I'm trying to set up a Nominatim 3.0.1 test installation and it seems to fail at [Installing Tiger housenumber data for the US](https://github.com/openstreetmap/Nominatim/blob/master/docs/Import-and-Update.md#installing-tiger-housenumber-data-for-the-us) whenever I try to import 2017 data while 2016 and 2015 (and probably earlier as well but untested) work as expected.

### 2015
```
Processing 01001...
```
### 2016
```
Processing 01001...
```
### 2017
```
Processing 01001...
Traceback (most recent call last):
  File "/data/nominatim/Nominatim/utils/tigerAddressImport.py", line 4162, in <module>
    shape_to_osm( shape, osm, id )
  File "/data/nominatim/Nominatim/utils/tigerAddressImport.py", line 4053, in shape_to_osm
    parsed_features = parse_shp_for_osm( shp_filename )
  File "/data/nominatim/Nominatim/utils/tigerAddressImport.py", line 3564, in parse_shp_for_osm
    divroad = poFeature.GetField("DIVROAD")
  File "/usr/lib/python2.7/dist-packages/osgeo/ogr.py", line 2845, in GetField
    raise ValueError("Illegal field requested in GetField()")
ValueError: Illegal field requested in GetField()
Failed parse (/data/nominatim/tiger/tl_2017_01001_edges.zip)
```

Is this supposed to work and has this been successfully tested already or should I rather downgrade to using data from 2016 or 2015 instead?

**Comment by lonvia** [comment]
Trying out the 2017 data is on the TODO list but a fix for the new format will then only appear in next release. For 3.0.1 you should use the 2016 data.

**Comment by mtmail** [comment]
The `DIVROAD` is no longer in the shapefile field list. Doublechecked the TIGER documentation. I need to cleanup my pull request which works on both 2016 and 2017 data. Alternatively you could comment out `poFeature.GetField("DIVROAD")` and the next couple of lines in `utils/tigerAddressImport.py`

**Comment by lonvia** [comment]
Fixed by #819.
